packages/phoenix-client/src/phoenix/client/resources/spans/__init__.py

```diff
@@ -808,6 +913,73 @@ async def get_spans(
 
         return all_spans[:limit]
 
+    async def log_spans(
+        self,
+        *,
+        project_identifier: str,
+        spans: Sequence[v1.Span],
```

Comment:
how easy is it to construct a v1.Span?

Reply:
currently we get them from either `get_spans` or converting them from dataframes. Constructing them requires using our typedict for autocomplete, which can be a bit cumbersome. The most ergonomic path, currently is to have loaded a collection of v1.Spans from spans that were previously retrieved and saved as JSON